man/man1/kbsecret-rm.1.ronnpp

```diff
@@ -3,24 +3,24 @@ kbsecret-rm(1) - remove a kbsecret(1) record
 
 ## SYNOPSIS
 
-`kbsecret rm` [options] <record>
+`kbsecret rm` [options] <record ![record ...]>
 
 ## DESCRIPTION
 
-`kbsecret rm` is the opposite of kbsecret-new(1) - it deletes the requested record.
+`kbsecret rm` is the opposite of kbsecret-new(1) - it deletes the requested record(s).
 
 ## OPTIONS
 
 * `-s`, `--session` <session>:
-	The session that the record is under.
+	The session that the record(s) are under.
 
 * `-i`, `--interactive`:
 	Confirm deletion interactively.
 
 ## EXAMPLES
 
 ```
-	$ kbsecret rm gmail
+	$ kbsecret rm gmail youtube
```

Comment:
Instead of changing the example, add a second example so that people understand that they can pass either one or many records to `kbsecret rm`.